source/2020-08-14-the-ember-times-issue-160.md

```diff
@@ -9,21 +9,36 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+"I contribute to Ember" with Isaac Lee and Jared Galanis 🙌,
 How to participate in the Ember RFC process 🚀,
 EmberFest: Share your virtual Hello with us! 🐹,
 
 READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## ["I contribute to Ember" with Isaac Lee and Jared Galanis 🙌](https://discuss.emberjs.com/t/i-contribute-to-ember-with-isaac-lee-and-jared-galanis/18127)
 
-<change section title emoji>
 <consider adding some bold to your paragraph>
 <please include link to external article/repo/etc in paragraph / body text, not just header title above>
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+<div class="float-right padded portrait-frame">
```

Comment:
I'm pretty sure the 2x image markup won't work as-is, but putting in as a placeholder